rfcs/0143-nix-store-acls.md

```diff
@@ -0,0 +1,195 @@
+---
+feature: nix-store-acls
+start-date: 2023-01-16
+author: Alexander Bantyev
+co-authors: Silvan Mosberger, Théophane Hufschmitt
+shepherd-team: (names, to be nominated and accepted by RFC steering committee)
+shepherd-leader: (name to be appointed by RFC steering committee)
```

Comment:
```suggestion
shepherd-team: John Ericson, Théophane Hufschmitt, Eelco Dolstra
shepherd-leader: Théophane Hufschmitt
```
@thufschmitt are you willing to take the lead on this RFC?

Reply:
Gladly :)